iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalGrammar.java

```diff
@@ -45,13 +46,15 @@ public enum BicepLexicalGrammar implements GrammarRuleKey {
    * Statements
    */
   STATEMENT,
+  IDENTIFIER,
```

Comment:
Can identifier be a statement on its own? Maybe it should be moved to expressions?

Reply:
No, an identifier can't be a statement on its own.
Let's move it.

Reply:
IMO those comments don't have much value, sometimes it's hard to decide what is what.